server/api/observation/type/observation-type-create.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response } from 'express';
+import { ObservationType } from './observation-type.model';
+
+class ObservationTypeCreateController {
+  async createObservationType(req: Request, res: Response) {
+    return res.json(await ObservationType.save(toObservationType(req.params.type)));
```

Comment:
Instead of the query parameter used JSON body.

Reply:
In theory, yes it would be a `body`, but going back to my previous comment... I wonder if we need ObservationType at all.

Reply:
Thanks! This is also mentioned below so I am closing this.